scripts/orca-singularity

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+if [ "$1" = "-c" ];then
+        cmd=`echo $2 | grep "rsync\|scp"`
+        if [ -z "$cmd" ]; then
+                echo "======================================================="
+                echo "           Welcome to BCGSC ORCA Service          "
+                echo "Only direct rsync or scp file transfer commands allowed"
+                echo "======================================================="
+        else
+                exec $2
+        fi
+else
+        USER=$(id -un)
```

Comment:
Unless there's a particular reason for this line, I'd remove it. `$USER` is already defined.